README.txt

```diff
@@ -1,36 +1,48 @@
+# vbap
 
-A Pd port of the VBAP object for Max/MSP
+A Pure Data port of the VBAP object for Max/MSP
 
-Written by Ville Pulkki
-Helsinki University of Technology
-Laboratory of acoustics and audio signal processing
+Originally written by Ville Pulkki  
+Helsinki University of Technology  
```

Comment:
nitpick, but of course i would strip the trailing whitespace.

Reply:
Done.